docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>I think it would be good to add a little bit more of an explanation of how the eye movement relates to the plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I vaguely remember reading eye movement somewhere but I am not able to find it in the paper. Any idea where this is explained?</p>

Reply:
<p>I don't know if this is the same but here's what I've read before in another similar dataset: https://stumpy.readthedocs.io/en/latest/Tutorial_Consensus_Motif.html?highlight=eye#Loading-the-Eye-tracking-(EOG)-Dataset</p>

Reply:
I wasn't able to find anything about it being eye-tracking or EOG, but the language in the paper seemed to me to indicate that it was pen position so I went with that